<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameCallClass</class>
 <widget class="QMainWindow" name="GameCallClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>484</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>GameCall</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>30</y>
      <width>251</width>
      <height>411</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="aboutBtn">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>450</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="text">
     <string>关于</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="text">
     <string>基址：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="text">
     <string>偏移：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>180</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="text">
     <string>调用有参数的Call代码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>70</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="text">
     <string>修改或者获取</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="text">
     <string>要修改的值：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>71</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="text">
     <string>要获取的值：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>70</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="callBtn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>450</y>
      <width>251</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="text">
     <string>调用Call</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="text">
     <string>进程名字：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pidFreshBtn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="text">
     <string>刷新</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="text">
     <string>当前选中程序：</string>
    </property>
   </widget>
   <widget class="QLabel" name="clickTipslabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>450</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="text">
     <string>暂无（双击附加进程）</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="CKWidthEdit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>100</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>101</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="text">
     <string>宽：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>101</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="text">
     <string>高：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="CKHeightEdit">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>100</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="changeCKSizeBtn">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>100</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="text">
     <string>修改窗口大小</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshSizeButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>130</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>恢复窗口大小</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>160</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="text">
     <string>调用无参数的Call</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="callAddresEdit">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>160</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="placeholderText">
     <string>请输入Call地址</string>
    </property>
   </widget>
   <widget class="QPushButton" name="callSignButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>160</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="text">
     <string>调用Call</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="callTextEdit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>210</y>
      <width>331</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="GameCall.qrc"/>
 </resources>
 <connections/>
</ui>
